geps/gep-1324.md

```diff
@@ -164,3 +164,11 @@ These are goals that we aren’t explicitly excluding, but will reconsider at a
 * Multicluster use-cases (e.g. Kubernetes MCS or cluster federation)
 * Heterogenous deployment targets (e.g VMs, serverless)
 * Deployment models (e.g. installation, mesh upgrades)
+
+## Graduation Criteria
+
+The following are items that **MUST** be resolved to move this GEP to
+`Standard` status (and before the end of the probationary period):
+
+...
+
```

Comment:
Assuming this particular part is WIP, just leaving a comment so we don't accidentally merge before we have content here.

Reply:
This is fixed.